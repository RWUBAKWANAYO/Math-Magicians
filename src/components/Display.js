import React from 'react';
import PropTypes from 'prop-types';

const Display = (props) => {
  const { total, next } = props;
  return (
    <div className="calculator-output">{next || total || 0}</div>
  );
};
Display.propTypes = {
  next: PropTypes.string,
  total: PropTypes.string,
};
Display.defaultProps = {
  next: null,
  total: null,
};

export default Display;
