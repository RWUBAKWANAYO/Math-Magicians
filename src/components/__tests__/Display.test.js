import renderer from 'react-test-renderer';
import Display from '../Display';

it('Display component render', () => {
  const tree = renderer.create(<Display />).toJSON();
  expect(tree).toMatchSnapshot();
});
