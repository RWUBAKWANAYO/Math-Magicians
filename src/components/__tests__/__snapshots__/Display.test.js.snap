// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component render 1`] = `
<div
  className="calculator-output"
>
  0
</div>
`;
